product-service/functions/getProductById.ts

```diff
@@ -11,8 +11,10 @@ const baseResponse = {
 export const getProductById: APIGatewayProxyHandler = async (event, _context) => {
     const { productId } = event.pathParameters;
 
+    console.log('CALL getProductById with productId: ', productId);
+
     try {
-        const product = await getById(productId);
+        const product = await productRepository.getById(productId);
 
         if (product === undefined) {
```

Comment:
It is enough to check if (product) or if (!!product). It is better to return statusCode 400

Reply:
Rewrote condition. Сoncerning statusCode = 400, I don't think so. In this case, we send 404 because we product not found. 400 errors we use for errors with incorrect parameters in request